File: Assets/Scripts/Main/PaperUIText.js
#pragma strict

import SimpleJSON;
import UnityEngine.UI;

function Start () {
	var data : TextAsset = Resources.Load.<TextAsset>("Stories");
	var json : JSONNode = JSON.Parse(data.text);
	var stories : JSONArray = json["Stories"].AsArray;
	
	var index : int = PlayerPrefs.GetInt("NowStageIndex");
	GetComponent(Text).text = stories[index]["Content"];
}

File: Assets/Scripts/Main/PaperUIText.cs
using UnityEngine.UI;
using UnityEngine;
using System.Collections;

[System.Serializable]
public partial class PaperUIText : MonoBehaviour
{
    public virtual void Start()
    {
        int index = PlayerPrefs.GetInt("NowStageIndex");
		GetComponent<Text>().text = Global.stories[index].content;
    }

}